.gitignore

```diff
@@ -26,26 +26,18 @@ build/
 
 # for a library or gem, you might want to ignore these files since the code is
 # intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
-# .ruby-version
-# .ruby-gemset
+Gemfile.lock
```

Comment:
Why adding `Gemfile.lock`, as far as I know I shouldn't commit this file to git.